File: codes/PostCalculator/Java/PostCalculator/PostCalculatorMain.java
package PostCalculator;

import java.util.Arrays;

public class PostCalculatorMain {
    public static void main(String[] args) {
        String postExp1 = "42*8+";
        String postExp2 = "123+*4/";

        System.out.println(postExp1+" = "+PostCalculator.EvalRPNExp(postExp1.toCharArray()));
        System.out.println(postExp2+" = "+PostCalculator.EvalRPNExp(postExp2.toCharArray()));
    }
}

File: codes/PostCalculator/Python/PostCalculatorMain.py
import PostCalculatorModule as pcm

postExp1 = "42*8+"
postExp2 = "123+*4/"

print("{} = {}".format(postExp1, pcm.PostCalculator.EvalRPNExp(postExp1)))
print("{} = {}".format(postExp2, pcm.PostCalculator.EvalRPNExp(postExp2)))
